lab-matt/test/index-test.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+var main = require(__dirname + '/../index');
```

Comment:
You're not being graded on this for this assignment but generally you'll want to be testing your /lib files directly, not through index.js.  Looks good though, just a note to keep in mind :)